src/wazuh_modules/inventory_harvester/src/systemInventory/elements/hotfixElement.hpp

```diff
@@ -13,8 +13,9 @@
 #define _OS_HOTFIX_HPP
 
 #include "../../wcsModel/data.hpp"
-#include "../../wcsModel/inventoryHotfixHarvester.hpp"
+#include "../../wcsModel/inventoryPackageHarvester.hpp"
```

Comment:
question: Hotfixes are "packages" now?

Reply:
yes, check the mapping!

https://github.com/wazuh/wazuh/blob/2967ecc4ef296ec0218947997343949d3fb7b6df/src/wazuh_modules/inventory_harvester/indexer/template/wazuh-states-inventory-hotfixes.json#L53

Reply:
Thanks!